### PR TITLE
Expose particle status client option

### PR DESCRIPTION
**Pull request by Lulu13022002**

### CODE REVIEW

File: patches/api/0183-Add-Player-Client-Options-API.patch [hunk 1]
```diff
@@ -60,6 +61,24 @@ index 0000000000000000000000000000000000000000..ed08f823e0620289392f7fc2ff0ac721
 +            return "options.chat.visibility." + this.name;
 +        }
 +    }
++
++    public enum ParticleStatus implements Translatable {
```
Comment (MiniDigger) [inline patches/api/0183-Add-Player-Client-Options-API.patch]: is ParticleStatus a good name for this? maybe ParticleVisibility?

Comment (Lulu13022002) [reply patches/api/0183-Add-Player-Client-Options-API.patch]: Yes ParticleVisibility is more explicit, maybe ParticleAmount too idk.
Do you remember when ChatVisibility is unknown? the field doesn't seems to be nullable and is init in the constructor

Comment (MiniDigger) [reply patches/api/0183-Add-Player-Client-Options-API.patch]: cant look at code rn, but theres a time period where the client options arent known to the server yet, right?

Comment (Lulu13022002) [reply patches/api/0183-Add-Player-Client-Options-API.patch]: Yes but in that case the player just have the default settings value. So we can't really say when the option is unknown. For example in the PlayerLoginEvent chat visibility is always full.

**Review by Machine-Maker — APPROVED** [review]

**Review by lynxplay — APPROVED** [review]

**Pull request merged**